src/scripts.js

```diff
@@ -31,17 +31,21 @@ let currentView;
 
 //event listeners
 recipeSection.addEventListener('click', createRecipeModal);
+recipeSection.addEventListener('keypress', createRecipeModal);
 modalSection.addEventListener('click', collapseRecipe);
 filterDropdown.addEventListener('click', filterRecipes);
 navMyRecipes.addEventListener('click', showSavedRecipes);
+navMyRecipes.addEventListener('keypress', (event) => {
+    if(event.key === 'Enter') showSavedRecipes()});
 logo.addEventListener('click', goHome);
+logo.addEventListener('keypress', goHome);
```

Comment:
This piece needs this: "('keypress', (event) => {
    if(event.key === 'Enter') goHome()})" added to the event listener.